tests/integration/tabular/test_base.py

```diff
@@ -78,7 +78,7 @@ def test_conditional_sampling_graceful_reject_sampling_True_dict(model):
         })
     ]
 
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError):  # noqa: PT011
```

Comment:
This can crash with different errors:

1) For GaussianCopula: `cannot reshape array of size 0 into shape (0)`
2) For other models: `Unable to sample any rows for the given conditions. Try increasing max_tries_per_batch (currently: 100). Note that increasing this value will also increase the sampling time.` 

Not sure if this is expected, I'll investigate. Regardless, the fix shouldn't be part of this PR, I think.